### ISSUE TITLE
 java.lang.RuntimeException: Package manager has died occurred in app invoke SDK for Android version 6.0.1(marshmello)

### ISSUE THREAD

**Issue by EffyCoder** [mrdishant/Paytm-All-in-one-flutter-sdk#6]
`
I/in.appyflow.paytm.PaytmPlugin( 8603): com.paytm.pgsdk.PaytmOrder@209c742
E/JavaBinder( 8603): !!! FAILED BINDER TRANSACTION !!!  (parcel size = 124)
E/MethodChannel#paytm( 8603): Failed to handle method call
E/MethodChannel#paytm( 8603): java.lang.RuntimeException: Package manager has died
E/MethodChannel#paytm( 8603): 	at android.app.ApplicationPackageManager.getPackageInfo(ApplicationPackageManager.java:135)
E/MethodChannel#paytm( 8603): 	at com.paytm.pgsdk.TransactionManager.getPaytmVersion(TransactionManager.java:98)
E/MethodChannel#paytm( 8603): 	at com.paytm.pgsdk.TransactionManager.startTransaction(TransactionManager.java:68)
E/MethodChannel#paytm( 8603): 	at in.appyflow.paytm.PaytmPlugin.beginPayment(PaytmPlugin.java:216)
E/MethodChannel#paytm( 8603): 	at in.appyflow.paytm.PaytmPlugin.onMethodCall(PaytmPlugin.java:98)
E/MethodChannel#paytm( 8603): 	at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:230)
E/MethodChannel#paytm( 8603): 	at io.flutter.embedding.engine.dart.DartMessenger.handleMessageFromDart(DartMessenger.java:85)
E/MethodChannel#paytm( 8603): 	at io.flutter.embedding.engine.FlutterJNI.handlePlatformMessage(FlutterJNI.java:692)
E/MethodChannel#paytm( 8603): 	at android.os.MessageQueue.nativePollOnce(Native Method)
E/MethodChannel#paytm( 8603): 	at android.os.MessageQueue.next(MessageQueue.java:323)
E/MethodChannel#paytm( 8603): 	at android.os.Looper.loop(Looper.java:136)
E/MethodChannel#paytm( 8603): 	at android.app.ActivityThread.main(ActivityThread.java:5497)
E/MethodChannel#paytm( 8603): 	at java.lang.reflect.Method.invoke(Native Method)
E/MethodChannel#paytm( 8603): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
E/MethodChannel#paytm( 8603): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
E/MethodChannel#paytm( 8603): Caused by: android.os.DeadObjectException: Transaction failed on small parcel; remote process probably died
E/MethodChannel#paytm( 8603): 	at android.os.BinderProxy.transactNative(Native Method)
E/MethodChannel#paytm( 8603): 	at android.os.BinderProxy.transact(Binder.java:536)
E/MethodChannel#paytm( 8603): 	at android.content.pm.IPackageManager$Stub$Proxy.getPackageInfo(IPackageManager.java:2202)
E/MethodChannel#paytm( 8603): 	at android.app.ApplicationPackageManager.getPackageInfo(ApplicationPackageManager.java:130)
E/MethodChannel#paytm( 8603): 	... 14 more

 `
>
E/flutter ( 8603): [ERROR:flutter/lib/ui/ui_dart_state.cc(166)] Unhandled Exception: PlatformException(error, Package manager has died, null)
E/flutter ( 8603):       StandardMethodCodec.decodeEnvelope (package:flutter/src/services/message_codecs.dart:572:7)
E/flutter ( 8603):       MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:161:18)
E/flutter ( 8603): <asynchronous suspension>
E/flutter ( 8603):       MethodChannel.invokeMethod (package:flutter/src/services/platform_channel.dart:334:12)
E/flutter ( 8603):       Paytm.payWithPaytm (package:paytm/paytm.dart:22:24)
E/flutter ( 8603):       _WalletPageState.build.<anonymous closure> 
> 

@mrdishant may be paytm has solved this issue in their latest appinvoke sdk.

**Comment by EffyCoder** [comment]
Hi @mrdishant Paytm has given below reason, 

```
As per our internal team,
Please check the minSdkVersion and use the above the minSdkVersion 18. Google no longer supports less than minSdkVersion 18 for security patches.  
```
This plugin uses minSDKVersion =18, it should be updated to 19. Also, update the plugin to use their latest appinvoke sdk for android. Thank you very much for this plugin.

**Comment by mrdishant** [comment]
Okay @EffyCoder 

I ll update the plugin asap

**Comment by mrdishant** [comment]
@EffyCoder 
Plugin Updated. Let me know its resolved or not

**Comment by mrdishant** [comment]
Closing the issue as updated plugin resolved this issue.

**Comment by EffyCoder** [comment]
Hi @mrdishant It didn't resolve the issue. We are taking follow up from paytm team meanwhile they have asked us to use appinvokesdk : 1.4 to see if the issue still persists.

**Comment by EffyCoder** [comment]
> Hi @mrdishant It didn't resolve the issue. We are taking follow up from paytm team meanwhile they have asked us to use appinvokesdk : 1.4 to see if the issue still persists. I'll update here if we get any solution.

There was an issue and it's fixed with appinvokesdk 1.4.

**Comment by mrdishant** [comment]
@EffyCoder Plugin updated to 1.4
